docs/dg/dev/system-requirements/202407.0/system-requirements.md

```diff
@@ -0,0 +1,55 @@
+| Databases                                 | Depending on the project, one of the databases: MariaDB >= 10.4—preferred, PostgreSQL >=9.6, or MySQL >=5.7.                                                                                                                                                                                                                                                                                                   |
+| PHP                                       | Spryker supports PHP `>=8.1` with the following extensions: `curl`, `json`, `mysql`, `pdo-sqlite`, `sqlite3`, `gd`, `intl`, `mysqli`, `pgsql`, `ssh2`, `gmp`, `mcrypt`, `pdo-mysql`, `readline`, `twig`, `imagick`, `memcache`, `pdo-pgsql`, `redis`, `xml`, `bz2`, and `mbstring`. For details about the supported PHP versions, see [Supported Versions of PHP](/docs/dg/dev/supported-versions-of-php.html) |
+| SSL                                       | For production systems, a valid security certificate is required for HTTPS.                                                                                                                                                                                                                                                                                                                                    |
+| Redis                                     | Version >=5.0, >=6.2                                                                                                                                                                                                                                                                                                                                                                                           |
+| Elasticsearch                             | Version 7.*x*                                                                                                                                                                                                                                                                                                                                                                                                  |
+| RabbitMQ                                  | Version 3.7+                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
Any reason to specifically 3.7 version?

Reply:
https://github.com/spryker/docker-sdk/tree/master/generator/src/templates/service/rabbitmq
Should we change it to `>=3.7` ?

Reply:
Nope, just was wondering why 3.7+, https://github.com/spryker/spryker-docs/pull/2604#discussion_r1560857689 answered, thanks